handler.go

```diff
@@ -161,7 +161,7 @@ func ReplyError(w http.ResponseWriter, err error) {
 	Reply(w, response, status)
 }
 
-// Helper that replies with the 500 code and happened error message.
+// Helper that replies 500 and message.
```

Comment:
Nitpick: the comment of a public entity should start with the name of that entity. `go lint` complains about things like that.